steering_vectors/aggregators.py

```diff
@@ -30,6 +52,33 @@ def pca_aggregator(pos_acts: Tensor, neg_acts: Tensor) -> Tensor:
     return sign * vec
 
 
+def _get_normalized_regression_coef(
+    pos_acts: Tensor,
+    neg_acts: Tensor,
+    regression: LinearRegression | LogisticRegression,
+) -> Tensor:
+    mean_acts = torch.stack([pos_acts, neg_acts]).mean(dim=0)
+
+    pos_acts_centered = pos_acts - mean_acts
+    neg_acts_centered = neg_acts - mean_acts
+
+    reg = regression.fit(
+        torch.cat([pos_acts_centered, neg_acts_centered])
+        .cpu()
+        .to(torch.float32)
+        .numpy(),
+        torch.cat([torch.ones(pos_acts.shape[0]), -1 * torch.ones(neg_acts.shape[0])])
+        .cpu()
+        .to(torch.float32)
+        .numpy(),
+    )
+
+    coef = torch.tensor([reg.coef_]).to(pos_acts.device).to(pos_acts.dtype)
```

Comment:
really minor, but this can be done in a single `.to()` call with `.to(pos_acts.device, pos_acts.dtype)` which might be slightly more efficient